### ISSUE TITLE
qsqlmysql.dll: The specified procedure could not be found

### ISSUE THREAD

**Issue by jonata** [thecodemonkey86/qt_mysql_driver#41]
Hey there! Thanks for this repository and your hard work. Can you help me find the source of my problem, if it's on my side or a bug?

Currently, when using version 6.6.2 on my project I get an intermitent error where data is not shown on `QTableView` . As far as I remember, I have no problem using version 6.6.1. Turns out I changed my computer/system, so I installed everything from scratch and version 6.6.1 stopped working. I did everything according to your documentation, the plugin is found and it tries to load. But I get this error:

> qt.core.library: "C:/Users/jbope/Projetos/XXXX/resources/plugins/sqldrivers/qsqlmysql.dll" cannot load: Cannot load library C:\Users\jbope\Projetos\XXXX\resources\plugins\sqldrivers\qsqlmysql.dll: The specified procedure could not be found.
qt.core.plugin.loader: QLibraryPrivate::loadPlugin failed on "C:/Users/jbope/Projetos/XXXX/resources/plugins/sqldrivers/qsqlmysql.dll" : "Cannot load library C:\\Users\\jbope\\Projetos\\XXXX\\resources\\plugins\\sqldrivers\\qsqlmysql.dll: The specified procedure could not be found."

> QSqlDatabase: QMYSQL driver not loaded

> QSqlDatabase: available drivers: QSQLITE QMIMER QODBC QPSQL QMARIADB QMYSQL

What could be the missing "procedure"? How can I check that?

I checked dependencies and it doesn't seem to be the problem:

![image](https://github.com/thecodemonkey86/qt_mysql_driver/assets/908065/67e543bd-0d79-460b-81c7-acf2421e8c34)

I tried every version of PySide6 6.*, with Python versions 3.8, 3.9, 3.10, 3.11.

I also tried to copy the DLLs to a common path like `windows/system32` , the project root and the python root, and nothing.

What am I missing?

Thanks for your attention.

**Comment by thecodemonkey86** [comment]
Hi,
just to make sure, are you using the MSVC variant of qsqlmysql.dll?

**Comment by jonata** [comment]
Yes. Windows 11, tested with Python versions 3.7, 3.8, 3.9, 3.10, 3.11 and 3.12.

**Comment by thecodemonkey86** [comment]
In the screenshot I see both paths Python310 and Python311. I'd recommend to fully uninstall Python and Qt, delete remaining files manually, also check PATH variable for remaining old entries, and then start fresh
